source/documentation/using_the_api/getting_updates.md

```diff
@@ -1,20 +1,103 @@
-## Updating registers
+## Getting updates
 
-You can find the latest entry number by looking at the register information (use the [`GET /register` endpoint](#get-register)) and comparing the most recent entry number with your own copy.
+You can use the [`GET /register/` endpoint](#getregister) to get the current highest entry number in a register from the `total-entries` property. You can then regularly compare it with new queries to the register to see if there are any updates.
 
-Example request: `https://local-authority-eng.register.gov.uk/register`
+### Return all entries since your last update
 
-Download a full new copy of the register by using [the `GET /download-register` endpoint](#get-download-register).
+You can use the [`GET /entries/` endpoint](#getentries) with the `start` and `limit` parameters to return specific entries to a register. 
 
-Download all updates for one record according to a particular key by [using the `GET /records/{key}/entries` endpoint](#get-records-key-entries):
+For example, the last highest entry number you got from the `country` register could be 205. You could then use the following request to find out if there are any new entries:
```

Comment:
We don't seem to specify anywhere how you know what the latest entry number you got was. This may have been removed from a previous version of this - did we deem it unnecessary?

Reply:
@arnau commented on this section in https://github.com/alphagov/registers-tech-docs/pull/93/files

☝️ that was the original PR for this content which I scrapped because too many changes were needed. 

> There is again the idea of knowing the highest entry number in my local copy which suggests to me that it should be one of the first recommendations in the "getting updates" section or at least the section that is about getting just the updates not the full set of records.

Originally this line:

> You can then regularly compare it with new queries to the register to see if there are any updates.

was this:

> You can keep track of this value and regularly compare it with new queries to the register to see if there are any updates.

("this value" means `total-entries`)